source/GameAction.cpp

```diff
@@ -462,6 +543,24 @@ void GameAction::Do(PlayerInfo &player, UI *ui, const Mission *caller) const
 			Audio::PlayMusic(music.value());
 	}
 
+	// Modify attributes.
+	for(auto &&it : modifyAttributes)
+	{
+		if(it.second)
+		{
+			DoModify(player, it.first, it.second);
+		}
+	}
+
+	// Set attributes.
+	for(auto &&it : setAttributes)
+	{
```

Comment:
We don't need these braces or the ones below
But if you like them you can keep them haha

Reply:
Honestly, I do like them. It's clearer and less ambiguous. Honestly, for a new coder looking at things, I think having the brackets makes it more legible.